arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/NonEmptyList.kt

```diff
@@ -241,8 +243,8 @@ public class NonEmptyList<out A>(
         "import arrow.core.toNonEmptyListOrNull"
       )
     )
-    @JvmStatic
-    public fun <A> fromListUnsafe(l: List<A>): NonEmptyList<A> =
+    @JvmStatic @JvmName("fromListUnsafe")
+    public inline fun <A> fromListUnsafe(l: List<A>): NonEmptyList<A> =
       NonEmptyList(l)
```

Comment:
Is there any good reason to keep this API? 🤔

Reply:
I think it's important to give a escape hatch in case somebody "knows what there're doing". But they could always use the version which may return null and do `!!` afterwards.

Reply:
> But they could always use the version which may return null and do !! afterwards.

Yes, that is actually what I was thinking of instead of explicitly exposing an API for it. We don't expose any escape hatches anywhere else in Kotlin since you can always quite easily explicitly walk around it in the few cases you need to.